apps/rush-lib/src/api/EnvironmentConfiguration.ts

```diff
@@ -67,6 +76,16 @@ export class EnvironmentConfiguration {
     return EnvironmentConfiguration._absoluteSymlinks;
   }
 
+  /**
+   * If this environment variable is set to "true", the Node.js version check will print a warning
+   * instead of causing a hard error if the environment's Node.js version doesn't match the
+   * version specifier in `rush.json`'s "nodeSupportedVersionRange" property.
+   */
```

Comment:
Add `See {@link EnvironmentVariableNames.RUSH_ALLOW_UNSUPPORTED_NODEJS}` like with the above one

Reply:
```suggestion
   *
   * See {@link EnvironmentVariableNames.RUSH_ALLOW_UNSUPPORTED_NODEJS}.
   */
```